packages/CodingStyle/src/Rector/Encapsed/EncapsedStringsToSprintfRector.php

```diff
@@ -74,7 +74,7 @@ public function refactor(Node $node): ?Node
             }
         }
 
-        $arguments = array_merge([new Arg(new String_($string))], $arguments);
+        array_unshift($arguments, new Arg(new String_($string)));
```

Comment:
I find array unshift unclear, does it un-drop something? 

Please keep former

Reply:
reverted :+1: